test/user/model.js

```diff
@@ -24,12 +24,12 @@ describe('<Unit Test>', function() {
         });
 
         describe('Method Save', function() {
```

Comment:
Itua who worked on integrating the test into Travis, confirmed the said test was failing due to the inability to connect to the database.
He requested the test be disabled for now - hence the commenting of the test case